playbook/app/pb_kits/playbook/pb_body/body.rb

```diff
@@ -18,16 +18,23 @@ class Body < Playbook::KitBase
                           default: false
       prop :highlighted_text, type: Playbook::Props::Array,
                               default: []
+      prop :truncate, type: Playbook::Props::Enum,
+                      values: [nil, "1", "2", "3", "4", "5"],
+                      default: nil
```

Comment:
If we're extending kits we need to add a doc and also make sure React is covered.